components/Buttons/src/MDCFloatingButton.h

```diff
@@ -60,17 +81,38 @@ typedef NS_ENUM(NSInteger, MDCFloatingButtonShape) {
 + (nonnull instancetype)floatingButtonWithShape:(MDCFloatingButtonShape)shape;
 
 /**
- @return The default floating button size dimension.
+ The mode of the floating button can either be .normal (a circle) or .extended (a pill-shaped
+ rounded rectangle).
+
+ The default value is @c .normal .
  */
-+ (CGFloat)defaultDimension;
+@property(nonatomic, assign) MDCFloatingButtonMode mode UI_APPEARANCE_SELECTOR;
 
 /**
- @return The mini floating button size dimension.
+ The position of the image relative to the title. Flipped for right-to-left.
+
+ The default value is @c .leading .
  */
-+ (CGFloat)miniDimension;
+@property(nonatomic, assign) MDCFloatingButtonImagePosition imagePosition UI_APPEARANCE_SELECTOR;
 
 /**
- Initializes self to a button with the given @c shape.
+ If @c YES, any values for @c contentEdgeInsets:forShape:mode: will be flipped when the FAB is
+ in @c MDCFloatingButtonModeExtended and its @c imagePosition is
+ @c MDCFloatingButtonImagePositionTrailing.
+
+ The default value is NO.
+ */
+@property(nonatomic, assign) BOOL contentEdgeInsetsFlippedForTrailingImagePosition
```

Comment:
I dont think this is useful.
We already have an api to specify a different contentEdge for the image position.

Reply:
Per offline-discussion, remove?

Reply:
@ianegordon @jverkoey @romoore discussed offline and we are going to remove this API.

Reply:
Done